workflows/amr/run.wdl

```diff
@@ -525,12 +521,10 @@ task ZipOutputs {
         mkdir ${TMPDIR}/outputs/raw_reports
         mkdir ${TMPDIR}/outputs/intermediate_files
 
-        counter=1
-        for fastx in ~{sep= ' ' nonHostReads}; do 
-            cp $fastx non_host_reads_R$counter."${fastx#*.}"
-            ((counter++))
-        done
-        cp ~{sep=' ' outputFiles} .
+        # copy contigs and interleave non_host_reads
+        cp ~{contigs_in} contigs.fasta
+        seqfu ilv -1 ~{sep=" -2 " nonHostReads} | gzip > non_host_reads.fasta.gz
```

Comment:
do I want to gzip this file?

Reply:
I think the unzipped version is what the frontend expects, but maybe it's a good question for product/compbio. I think the gzipped version would be a good space saving solution